src/js/tabs/coldwalletsettings.js

```diff
@@ -24,23 +24,6 @@ ColdwalletSettingsTab.prototype.angular = function(module) {
     if (!id.loginStatus) {
       id.goId();
     }
-    if ($scope.userBlob.data.defaultDirectory) {
```

Comment:
Is the plan here to remove the buttons from the UI so the user can't change the default directory?